internal/node.go

```diff
@@ -56,6 +56,7 @@ type Node struct {
 	// These are only accessible from the document root Node
 	Styles, Scripts    []*Node
 	HydratedComponents []*Node
+	ExternalResources  [][]Attribute
```

Comment:
I only wanted the `<link>` attributes, not necessarily entire tag context. Is this alright?